modules/network/traffic-manager/main.bicep

```diff
@@ -1,11 +1,34 @@
-@description('Prefix of Traffic Manager Profile Name')
-param prefix string = 'traf'
-
 @description('Name of Traffic Manager Profile Resource')
-param name string = '${prefix}-${uniqueString(resourceGroup().id, subscription().id)}'
+@minLength(1)
+@maxLength(63)
+param name string
 
 @description('Relative DNS name for the traffic manager profile, must be globally unique.')
-param trafficManagerDnsName string = 'tmp-${uniqueString(resourceGroup().id, subscription().id, name)}'
+param trafficManagerDnsName string
+
+@description('Tags for the module resources.')
+param tags object = {}
+
+@description('The traffic routing method of the Traffic Manager profile.')
```

Comment:
Can we add the default value to description?

Reply:
done!